web/app/themes/clarity/inc/admin/prior-party/prior-party-banner-email.php

```diff
@@ -80,30 +80,50 @@ public function timestampToLocalDateObject(int $timestamp): \DateTime
     /**
      * A timezone aware function that will send digest emails to those set in the Options page.
      * 
-     * @param array $props An array containing the DST status.
+     * @param ?array $props An optional array containing the DST status.
      * 
      * @return void
      */
 
-    public function maybeSendEmails($props)
+    public function maybeSendEmails(?array $props = []) : void
     {
         // Is London currently observing DST?
         $in_dst = strtotime("today 9:00 am Europe/London") !== strtotime("today 9:00 am UTC");
 
         // If the current DST status doesn't match the props, return early.
-        if ($in_dst !== $props['dst']) {
+        if (isset($props['dst']) && $props['dst'] !== $in_dst) {
             // It's 10am in the summer or 8am in the winter - don't send emails.
             return;
         }
 
-        $users_to_email = get_field($this->user_field_name, 'option') ?: [];
-        $bcc_addresses_to_email = get_field($this->bcc_field_name, 'option') ?: [];
-        $all_recipients = array_merge($users_to_email, $bcc_addresses_to_email);
-
         $timestamp_from = strtotime('yesterday 9:00 Europe/London');
         $timestamp_to = strtotime('today 9:00 Europe/London');
         $email_content = $this->getEmailDigestByTimes($timestamp_from, $timestamp_to);
 
+        if(empty($email_content)) {
+            return;
+        }
+
+        // Was an email passed in manually - for testing purposes?
+        if(isset($props['recipient'])) {
+            wp_mail($props['recipient'], $email_content['subject'], $email_content['body']);
+            echo 'A test email was sent to email sent to ' . $props['recipient'];
+            // Return early.
+            return;
+        }
+
+        $all_recipients = [];
+        $users_to_email = get_field($this->user_field_name, 'option');
+        $bcc_addresses_to_email = get_field($this->bcc_field_name, 'option');
+
+        if(is_array($users_to_email)){
+            $all_recipients = array_merge($all_recipients, $users_to_email);
+        }
+
+        if(is_array($bcc_addresses_to_email)) {
+            $all_recipients = array_merge($all_recipients, $bcc_addresses_to_email);
+        }
+
         foreach ($all_recipients as $recipient) {
```

Comment:
Hey @EarthlingDavey the mail.php override function already manages an array of users - this might not be necessary

Reply:
You should be able to call like this:

```php
wp_mail($array_of_user_email_addresses, $email_content['subject'], $email_content['body']);
```

Reply:
Thanks, I've updated the code :)